### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bangzi1001**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+or price feed formula, logic and source. The BAIP should provide a concise technical specification of the
+feature or the idea and a rationale for it. It may not only describe technical
+improvements but also document *best-practises* and recommendations.
+
+The distinct document [BAIP Purpose and Guidelines](baip-0001.md) gives a more
+detailed explanation.
+
+# Available BSIPs
+
+Number             | Title                                                    | Owner             | Type           | Status
```
Comment (abitmore) [inline README.md]: I think it's better to update `Owner` to `Author`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
-# baips
-BitAssets Improvement Proposals
+# BitAssets Improvement Proposal
+
+BAIP stands for BitAssets Improvement Proposal but can also seen as an
+improvement *consensus*. BitAssets are BitShares Committee's SmartCoins
```
Comment (abitmore) [inline README.md]: `Committee's SmartCoins` doesn't sound accurate.

File: README.md (hunk 1 above)
Comment (bangzi1001) [reply README.md]: Change to Author

File: README.md (hunk 2 above)
Comment (bangzi1001) [reply README.md]: The BAIP description in forum sound better and change to it. 
https://bitsharestalk.org/index.php?topic=29723

**Review by abitmore — APPROVED** [review]

**Comment by abitmore** [comment]
Thanks!

**Pull request merged**